compiler/modules/front/stmt.bal

```diff
@@ -219,6 +233,7 @@ class StmtContext {
 function codeGenFunction(ModuleSymbols mod, s:FunctionDefn defn, bir:FunctionSignature signature) returns bir:FunctionCode|CodeGenError {
     StmtContext cx = new(mod, defn, signature.returnType);
     bir:BasicBlock startBlock = cx.createBasicBlock();
+    cx.openRegion(startBlock.label, "SIMPLE");
```

Comment:
Don't see why we need this region in the BIR.

Reply:
https://github.com/poorna2152/nballerina/issues/14#issuecomment-1063823574
The `Region#0` in the image in the above link is added by this function call.